README.md

```diff
@@ -69,7 +86,7 @@ spark-shell
   --conf spark.executor.extraClassPath=${gluten_jvm_jar} \
   --conf spark.shuffle.manager=org.apache.spark.shuffle.sort.ColumnarShuffleManager \
   ...
-  ```
+```
 
 ## 3.1 Build and Install Gluten with Velox backend
```

Comment:
the format is not correct here, it has two "3.1" bullet

Reply:
I will change 
"### 3.1 Build and Install Gluten with Velox backend" to “### 3.2.1 Build and Install Gluten with Velox backend”.